src/bandersnatch/main.py

```diff
@@ -152,6 +158,15 @@ def main():
         finally:
             loop.close()
     else:
+        if args.force_check:
+            status_file = os.path.join(config.get("mirror", "directory"), "status")
+            try:
+                shutil.move(status_file, "/tmp/status")
+                logger.debug(
+                    "Force bandersnatch to check everything against the master PyPI."
+                )
+            except Exception as e:
```

Comment:
Please don't Pokemon exception handle, (gotta catch em all) let us catch what we expect it to throw.
```suggestion
            except OSError as e:
```